packages/phishing-controller/package.json

```diff
@@ -44,8 +44,10 @@
   "dependencies": {
     "@metamask/base-controller": "^6.0.3",
     "@metamask/controller-utils": "^11.1.0",
+    "@noble/hashes": "^1.4.0",
     "@types/punycode": "^2.1.0",
     "eth-phishing-detect": "^1.2.0",
+    "ethereum-cryptography": "^2.1.2",
```

Comment:
The addition of these dependencies should be noted as `CHANGED` entries in the changelog, possibly as breaking changes.